tests/cases/fourslash/renameDefaultImportDifferentName.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path='fourslash.ts' />
 
 // @Filename: B.ts
-////export default class /*1*/C {
+////export default class /*1*/[|C|] {
```

Comment:
also add a test for `import * as m from "B"; m.default`; renaming should do nothing in this case.